### PR TITLE
ansible-test: bump acme test container version to 2.0.0

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Bumps the version of the ACME test container from 1.9.0 to 2.0.0. This change should not cause backwards compatibilities.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
ansible-test

### CODE REVIEW

**Comment by felixfontein** [comment]
@mattclay is it possible to backport this to stable-2.9 and stable-2.10? Or should I adjust the testing infrastructure in community.crypto to explicitly use this container with the `ANSIBLE_ACME_CONTAINER` environment variable?

**Comment by mattclay** [comment]
@felixfontein Is the new container backwards compatible with the older tests?

**Comment by felixfontein** [comment]
I think they should. I'm testing this in ansible-collections/community.crypto#98. If it works there, it will be fine for `devel` and `stable-2.10`, and pretty sure for `stable-2.9`. For `stable-2.9`, an attempted backport PR will show whether tests pass or not.

**Comment by mattclay** [comment]
As long as the container is backwards compatible I'm OK with backporting the change.

**Comment by felixfontein** [comment]
The container works fine with stable-2.10 and devel. The tests in the c.c PR passed, also with stable-2.9 ansible, and the tests in c.c are mostly identical to the ones in stable-2.9 (IIRC they test more features than the tests in stable-2.9).

File: changelogs/fragments/71097-ansible-test-acme-container.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- "ansible-test - the ACME test container was updated, it now supports external account creation and has a basic OCSP responder (https://github.com/ansible/ansible/pull/71097, https://github.com/ansible/acme-test-container/releases/tag/2.0.0)."
```
Comment (mattclay) [inline changelogs/fragments/71097-ansible-test-acme-container.yml]: Include a newline at the end of the file. Other than that, looks good.

**Review by mattclay — CHANGES_REQUESTED** [review]

File: changelogs/fragments/71097-ansible-test-acme-container.yml (hunk 1 above)
Comment (felixfontein) [reply changelogs/fragments/71097-ansible-test-acme-container.yml]: Fixed in c451823d4fc4f9da42cda5e0cc54d03b2e111609.

**Review by mattclay — APPROVED** [review]

**Comment by felixfontein** [comment]
@mattclay thanks for reviewing and merging!

**Pull request merged**